public/views/index.html

```diff
@@ -162,6 +163,21 @@
         display: block;
       }
 
+      .preloader--done .preloader__text--fail a {
+        color: white;
```

Comment:
This causes issues if you've been using the light theme (unclear when it is set). The theme is assigned to the body, so just `.light-theme` and add another color
![image](https://github.com/user-attachments/assets/8dd4b223-21e4-4f53-b350-50538947cad7)

Reply:
The orange doesn't pass the contrast check here either